main.tf

```diff
@@ -47,6 +47,7 @@ resource "aws_lambda_function" "this" {
   role             = aws_iam_role.this.arn
   handler          = "handler.run"
   runtime          = "python3.12"
+  timeout          = 60
```

Comment:
Imho, the 60 second timeout is an overkill, and it would be good to set it to lower value by default, in order to prevent buggy lambdas from running too long and wasting money. Making it configurable as you suggested in https://github.com/tradeparadigm/terraform-aws-ecr-repo-lambda/issues/7 would be best.

Reply:
How about setting it to 10 seconds, if you don't feel like updating the PR to make it configurable.

Reply:
I've done both - saves me coming back if 10s turns out to be too short :)